hcvault/keysource.go

```diff
@@ -128,12 +131,23 @@ func NewMasterKey(address, enginePath, keyName string) *MasterKey {
 	return key
 }
 
+// HTTPClient is a wrapper around http.Client used for configuring the
+// Vault client.
+type HTTPClient struct {
```

Comment:
This appears to be missing a constructor, rendering it unusable for external consumers.

Reply:
lol good catch sorry :facepalm: